template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl

```diff
@@ -44,21 +44,24 @@ to address any test failures.
 {{ if (eq .input_include_feature_store `yes`) }}
 The integration test runs the feature engineering and model training notebooks as a multi-task Databricks Job in the staging workspace.
 It reads input data, performs feature transforms, and writes outputs to Feature Store tables in the staging workspace. 
-The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in the model registry. 
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in 
+{{ if (eq .input_include_models_in_unity_catalog `no`) }} workspace model registry
+{{- else -}} UC
+{{end}}. 
 The fitted model along with its metrics and params will also be logged to an MLflow run. 
 To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run, which you can use
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
 to view training metrics or fetch and debug the model as needed.
 {{- else }}
 The integration test runs the model training notebook in the staging workspace, training, validating,
-and registering a new model version in the model registry. The fitted model along with its metrics and params
+and registering a new model version in 
+{{ if (eq .input_include_models_in_unity_catalog `no`) }} workspace model registry
+{{- else -}} UC
+{{end}}.
+The fitted model along with its metrics and params
 will also be logged to an MLflow run. To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run:
-
-![Link to MLFlow Run](images/MLFlowRunLink.png)
-
-Click the MLflow run link to view training metrics or fetch and debug the model as needed.
-
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
+to view training metrics or fetch and debug the model as needed.
```

Comment:
Also if we're removing the link to the `MLFlowRunLink.png` might as well delete the file

Reply:
Done!